code/game/objects/items/weapons/clown_items.dm

```diff
@@ -68,6 +68,13 @@
 	var/honk_sound = 'sound/items/bikehorn.ogg'
 	var/cooldowntime = 20
 
+/obj/item/weapon/bikehorn/syndie
+	desc = "Guaranteed to make you the subject of a Doctor Doctor joke."
+	attack_verb = list("VIOLENTLY HONKED", "VICIOUSLY MOCKED")
+	origin_tech = "combat=3;syndicate=1"
+	force = 25
+	no_embed = TRUE

```

Comment:
@Fox-McCloud some items still have it, which is why I added it when I saw it... I'll remove it from this PR, and submit another PR soon scrubbing the last references to `no_embed`